docs/src/main/asciidoc/datasource.adoc

```diff
@@ -229,16 +233,34 @@ As previously stated, most of the time, this automatic resolution will suit you
 you won't need to configure the driver.
 ====
 
-Note that you can still define a specific driver if you need to (for instance for using the OpenTracing driver)
-or if you want to use a database for which we don't have a built-in JDBC driver extension.
+[[other-databases]]
+==== Use a database with no built-in extension or with a different driver
+
+You can use a specific driver if you need to (for instance for using the OpenTracing driver)
+or if you want to use a database for which Quarkus does not have a built-in JDBC driver extension.
 
-Defining a specific JDBC driver is as simple as:
+Without an extension, the driver will work fine in any Quarkus app running in JVM mode.
+It is unlikely that it will work when compiling your application to a native executable though.
```

Comment:
```suggestion
It is unlikely that it will work when compiling your application to a native executable.
```

Reply:
Why do you want to remove it? @machi1990 ? I understand my general rule is not to add such frivolous words in doc but here it's to balance the previous sentence